example_site/src/components/ConfigurationControls.js

```diff
@@ -218,14 +219,16 @@ export default function ConfigurationControls({ isOpen }) {
             opacity: partner === 'custom' ? 1 : 0
           }}
         >
-          <div>
-            <h2>Configuration</h2>
+          <div className="configuration-controls-subsection"> 
+            <div className="setting-title">Configuration</div>
 
             <label title="Environment" className="form-label" htmlFor="environment">
               Environment Connection
             </label>
             <select
               name="environment"
+              className="select-input"
+              style = {{width: '45%'}}
```

Comment:
Two things - you've got some extra whitespace here, but why are we using a style object instead of CSS? It doesn't look like this value needs to be dynamic at all

Reply:
`className="select-input new-class"`